src/Inventory/Asset/NetworkEquipment.php

```diff
@@ -80,7 +80,8 @@ public function prepare(): array
                 'model'        => $model_field,
                 'type'         => $types_field,
                 'manufacturer' => 'manufacturers_id',
-                'credentials'  => 'snmpcredentials_id'
+                'credentials'  => 'snmpcredentials_id',
+                'ip'           => 'remote_addr'
```

Comment:
Are we sure existing IP in very old XML file is the contact address we now want to handle? 
I guess the json scheam should be changed again to add a `remote_addr` node, that would be more self eplainnign than `ip`.

Reply:
done with -> https://github.com/glpi-project/inventory_format/pull/91